config/configuration.toml.in

```diff
@@ -1,6 +1,6 @@
 # XXX: Warning: this file is auto-generated from file "@CONFIG_IN@".
 
-[hypervisor.qemu-lite]
+[hypervisor.q35]
```

Comment:
@sboeuf indeed, I don't think we've settled on a name for the q35 qemu - and we should probably call it something else.
I was a little unsure with it living in the 'pc-lite' branch in the repo even.
I can see it is a 'progression of pc-lite', but having the same name will make some of our integrations hard. /cc @anthonyzxu and @mcastelino and @egernst for their thoughts here.

(to note - I renamed mine locally when I was hand testing ;-) )